### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by BansodeSahithiDeshmukh** [aldabil21/react-scheduler#171]
Hi

Although there is no direct option/prop to make it possible, it is doable by doing the following:

1. Use `onConfirm` and make sure you validate the add/edit functionality, if there is an event in the same slot, the intercept and return false. Another option is, you can use `customEditor` to make your own form/fields and also validate before save. However, validation should be in your backend to prevent saving 2 events in the same slot.
2. Use `onEventDrop` top intercept dropping event and validate if the slot already has an event, then return void/false.

Feel free to re-open if you still have an issue.

_Originally posted by @aldabil21 in https://github.com/aldabil21/react-scheduler/issues/124#issuecomment-1373634569_

**Comment by aldabil21** [comment]
Here: https://codesandbox.io/s/171-9i4h8c

Though, you can use the same concept with a `customEditor`, in order to display more _pretty_ validation error, since you will have full control of your own input fields.

Is this helpful to you?

**Comment by BansodeSahithiDeshmukh** [comment]
<img width="944" alt="image" src="https://user-images.githubusercontent.com/123079502/228756964-c36e8425-610e-441a-ac4f-876bcac843e9.png">

in the above mentioned code sandbox  i am still able see the events in the same slot

**Comment by aldabil21** [comment]
The sandbox code won't let you add 2 events in the same slot.
but if you put them manually in the `events` array, then it will show of course.

Your exactly issue is "How to not add events in the time slot", which is related to event add/edit time __validation__, that has nothing to do with this Scheduler component. 

If you have don't control the source of the events array (which mean you don't control adding/editing validation process), then you should loop through the events array and filter out the event that are intersecting, before feeding the `events` to the `scheduler` component.

Basically, its a validation thing in your add/edit/fetch process. you have to do that your self
